documentation/manual/working/javaGuide/main/async/code/javaguide/async/JavaStream.java

```
@@ -125,7 +124,9 @@ public void serveFile() throws Exception {
         File file = new File("/tmp/fileToServe.pdf");
         file.deleteOnExit();
         try (OutputStream os = java.nio.file.Files.newOutputStream(file.toPath())) {
-            IOUtils.write("hi", os, "UTF-8");
+            os.write("hi".getBytes("UTF-8"));
+        } catch (IOException e) {
```

actually `IOUtils.write` will silently ignore the error, but I do not think that this is a good idea to have in our docs.